modules/n1ql/examples/dml/delete-offset-before.jsonc

```diff
@@ -0,0 +1,22 @@
+id
```

Comment:
A bit of a nit - the "Results" do not match the format of the results returned by its corresponding example query.

Reply:
Agreed, though that is what gets copied when I use the copy icon and paste in -- it is odd that the format is different, but it certainly takes up less space... 
what would be your preference here (and below), @simon-dew ?

Reply:
after dropping travel-sample and reinstalling it, I have redone both sets of results